model.py

```diff
@@ -4795,24 +4776,34 @@ class CachedFeed(Base):
     # The content of the feed.
     content = Column(Unicode, nullable=True)
 
+    # A LicensePool for feeds associated with a certain work.
```

Comment:
I would say 'literary work' to avoid confusion with the Work data model object. Or just say 'LicensePool'.